recipe/meta.yaml

```diff
@@ -10,6 +10,8 @@ source:
     fn: opencv-{{ version }}.tar.gz
     sha256: 4eef85759d5450b183459ff216b4c0fa43e87a4f6aa92c8af649f89336f002ec
     patches:
+      # https://github.com/conda-forge/opencv-feedstock/pull/112
```

Comment:
Does this address this issue: https://github.com/conda-forge/opencv-feedstock/issues/41
If so, consider linking to it rather than this PR.

Reply:
Yes, it does.
Sure.